src/components/HelperTooltip/HelperTooltip.tsx

```diff
@@ -11,11 +12,11 @@ export type HelperTooltipProps = {
 
 export const HelperTooltip = ({ children, flexProps, iconProps, ...rest }: HelperTooltipProps) => {
   return (
-    <Tooltip {...rest}>
+    <TooltipWithTouch {...rest}>
```

Comment:
probs should memoize rest here too while you're at it?

Reply:
can also remove the spread here